index.js

```diff
@@ -15,20 +19,33 @@ const path = require("path");
 const db = require("./model/db");
 const verseInfo = require("./routes/verseInfo");
 const tafsir = require("./routes/tafsir");
+const authentication = require("./routes/auth");
+const { checkAuth } = require("./routes/utils");
 
-var port = 3001;
+const serviceAccount = require("./serviceAccountKey.json");
+
+admin.initializeApp({
```

Comment:
I'm not sure if this should be here, is it needed here or can it be moved to auth?

Reply:
we could, but not sure if that makes the most sense considering the current routes file structure. we can move it to routes/utils.js though.